apps/studio/components/interfaces/Auth/Hooks/HookCard.tsx

```diff
@@ -0,0 +1,108 @@
+import { Check, Webhook } from 'lucide-react'
+import { Badge, Button, Input } from 'ui'
+import { Hook } from './hooks.constants'
+
+interface HookCardProps {
+  hook: Hook
+  canUpdateConfig: boolean
+  onToggle: (enabled: boolean) => void
+  onSelect: () => void
+}
+
+export const HookCard = ({ hook, canUpdateConfig, onToggle, onSelect }: HookCardProps) => {
+  return (
+    <div className="bg-surface-100 border-default overflow-hidden border shadow px-5 py-4 flex flex-row">
+      <div className="px-3 pt-1">
```

Comment:
Maybe don't use paddin here.
Use padding on container div above and use gap better.

Reply:
This is how it looks without top padding:
![Screenshot 2024-05-27 at 13 32 09](https://github.com/supabase/supabase/assets/568291/cda51d2f-a551-4903-880e-aa657712a94b)

A bit weird to me, the icon being higher than the text.